src/panels/lovelace/cards/hui-todo-list-card.ts

```diff
@@ -319,10 +334,16 @@ export class HuiTodoListCard extends LitElement implements LovelaceCard {
                     "ui.panel.lovelace.cards.todo-list.no_unchecked_items"
                   )}
                 </p>`}
+            ${itemsWithoutStatus.length
+              ? html`
+                  <div class="divider" role="seperator"></div>
+                  ${this._renderItems(itemsWithoutStatus, unavailable)}
+                `
+              : nothing}
```

Comment:
Shouldn't this have a title? Also, if the list supports reordering and there are no active items the menu for reordering is currently not shown.